contrib/build-meep.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/bash
+
+# latest version of this script can be found at:
+#   https://github.com/NanoComp/meep/blob/master/contrib/build-meep.sh
+
+# detect if script is in a directory called src/
+DESTDIR=$(pwd)
```

Comment:
Might be nice to get this from an argument to the script rather than using `pwd`.

Reply:
That's true. It is now available.